operator/pkg/parsed/module.go

```diff
@@ -59,25 +55,12 @@ func (m *Module) StateMismatchedWithCondition(condition *v1alpha1.KymaCondition)
 		condition.TemplateInfo.Channel != m.Template.Spec.Channel
 }
 
-// UpdateModuleFromCluster update the module with necessary information (status, ownerReference) based on
-// an interaction with a client that is connected to a cluster. It will wrap any error returned from the client,
-// so checking for a k8s error can be achieved with Unwrap.
-func (m *Module) UpdateModuleFromCluster(ctx context.Context, clnt client.Client) error {
```

Comment:
Just want to mention, it's actually a hidden cyclic dependency.

Reply:
what I mean is previously by using this `UpdateModuleFromCluster`, it introduced `module` to `KymaReconciler`, and with this `client.Client` param, it potentially introduce `KymaReconciler` into `module`, which become cyclic depended, it' will not complain by the compiles although, since KymaReconciler used as Client interface, otherwise I believe it's cyclic import.